src/main/java/io/mosip/digitalcard/service/impl/SunbirdVCIServiceImpl.java

```diff
@@ -0,0 +1,138 @@
+package io.mosip.digitalcard.service.impl;
+
+import io.mosip.digitalcard.constant.ApiName;
+import io.mosip.digitalcard.controller.DigitalCardController;
+import io.mosip.digitalcard.dto.CreateInsuranceDto;
+import io.mosip.digitalcard.dto.CredentialStatusResponse;
+import io.mosip.digitalcard.dto.RegistrySearchRequestDto;
+import io.mosip.digitalcard.exception.DigitalCardServiceException;
+import io.mosip.digitalcard.service.SunbirdVCIService;
+import io.mosip.digitalcard.util.DigitalCardRepoLogger;
+import io.mosip.digitalcard.util.RestClient;
+import io.mosip.digitalcard.util.TokenIDGenerator;
+import io.mosip.digitalcard.util.Utility;
+import io.mosip.kernel.core.http.ResponseWrapper;
+import io.mosip.kernel.core.logger.spi.Logger;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * The SunbirdVCIServiceImpl.
+ *
+ * @author Dhanendra
+ */
+@Service
+public class SunbirdVCIServiceImpl implements SunbirdVCIService {
+
+    Logger logger = DigitalCardRepoLogger.getLogger(SunbirdVCIServiceImpl.class);
+
+    @Autowired
+    private TokenIDGenerator tokenIDGenerator;
+    @Value("${mosip.digitalcard.sunbird.vci.registry.url}")
+    private String registryURL;
+
+    @Value("${mosip.digitalcard.sunbird.vci.search.registry.url}")
+    private String searchRegistryURL;
+
+    @Value("${mosip.digitalcard.sunbird.vci.policy.Name}")
+    private String policyName;
+
+    @Value("${mosip.digitalcard.sunbird.vci.policy.expiry}")
+    private String policyExpiry;
+
+    @Value("${mosip.digitalcard.sunbird.vci.policy.issuedOn}")
+    private String policyIssuedOn;
+
+    @Value("${mosip.digitalcard.sunbird.vci.policy.id}")
+    private String policyId;
```

Comment:
can you rename this as mosip.digitalcard.sunbird.vci.policy.id-prefix and policyIdPrefix ?